### ISSUE TITLE
preprocess error

### ISSUE THREAD

**Issue by ninedow** [282857341/nnFormer#81]
Hi, thanks for your sharing!
I have a problem when preprocessing.
The error as follow:
nnFormer_plan_and_preprocess：未找到命令
Can someone tell me how to solve this problem? Thank you!

**Comment by 282857341** [comment]
Did you run the following commands?
```
source activate nnFormer
pip install -e .
```

**Comment by 282857341** [comment]
I guess you didn't run the commands in the environment of nnFormer

**Comment by ninedow** [comment]
Yes, I will have a try. Thank you very much!
And I have another problem. Training set has 80 patients in the ACDC_dataset.json.
 Is the training set : test set 8:2?

**Comment by 282857341** [comment]
No, actually this code is based on the foundation of nnunet. So in the Json file, the 'numTraining' contains both training nums and validation nums. You need to divide the training and validation set in the trainer

**Comment by ninedow** [comment]
OK，Thank you very much！！
